database-commons/src/main/java/io/cdap/plugin/db/batch/sink/AbstractDBSink.java

```diff
@@ -141,8 +143,14 @@ public void prepareRun(BatchSinkContext context) {
     configAccessor.setInitQueries(dbSinkConfig.getInitQueries());
     configAccessor.getConfiguration().set(DBConfiguration.DRIVER_CLASS_PROPERTY, driverClass.getName());
     configAccessor.getConfiguration().set(DBConfiguration.URL_PROPERTY, connectionString);
-    configAccessor.getConfiguration().set(DBConfiguration.OUTPUT_TABLE_NAME_PROPERTY,
-                                          dbSinkConfig.getEscapedTableName());
+    if (dbSinkConfig.dbSchemaName == null) {
+      configAccessor.getConfiguration().set(DBConfiguration.OUTPUT_TABLE_NAME_PROPERTY,
+            dbSinkConfig.getEscapedTableName());
+    }
+    if (dbSinkConfig.dbSchemaName != null) {
+      configAccessor.getConfiguration().set(DBConfiguration.OUTPUT_TABLE_NAME_PROPERTY,
+              dbSinkConfig.getEscapedSchemaName() + "." + dbSinkConfig.getEscapedTableName());
+    }
```

Comment:
minor nit:
you can remove these if conditions by doing something similar to 
```java
    String fullyQualifiedTableName = dbSinkConfig.dbSchemaName == null ? dbSinkConfig.getEscapedTableName() 
      : dbSinkConfig.getEscapedSchemaName() + "." + dbSinkConfig.getEscapedTableName();
    configAccessor.getConfiguration().set(DBConfiguration.OUTPUT_TABLE_NAME_PROPERTY, fullyQualifiedTableName);
```

Reply:
way better way of writing it. made change.